holidays/countries/mongolia.py

```diff
@@ -0,0 +1,97 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import InternationalHolidays, MongolianCalendarHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Mongolia(HolidayBase, InternationalHolidays, MongolianCalendarHolidays):
+    """Mongolia holidays.
+
+    References:
+        * <https://en.wikipedia.org/wiki/Public_holidays_in_Mongolia>
+        * <https://www.timeanddate.com/holidays/mongolia>
+        * <https://investmongolia.gov.mn/mongolia-at-a-glance/>
+        * <https://www.math.mcgill.ca/gantumur/cal/year.html>
```

Comment:
```suggestion
        * [Law on Public holidays and days of observation](https://web.archive.org/web/20250327062440/https://legalinfo.mn/mn/detail/399)
        * [Labor Law](https://web.archive.org/web/20250421093230/https://legalinfo.mn/mn/detail?lawId=16230709635751)
        * <https://en.wikipedia.org/wiki/Public_holidays_in_Mongolia>
        * [Mongolian lunar calendar](https://web.archive.org/web/20230412171012/https://www.math.mcgill.ca/gantumur/cal/index_mn.html)
        * <https://www.timeanddate.com/holidays/mongolia>
        * <https://investmongolia.gov.mn/mongolia-at-a-glance/>
```
These laws (especially the first one) are our main source of information. I strongly advise you to review and correct the original names of the holidays and the years of their introduction.